source/clear-linux/guides/telemetrics/telem-guide.rst

```diff
@@ -3,7 +3,7 @@
 Telemetrics
 ###########
 
-Telemetrics in |CL-ATTR| is a client and server solution used to collect data from running |CL| systems to help quickly identify and fix bugs in the OS.  Both client and server are customizable, and an API is available on the client side for instrumenting your code for debug and analysis.
+Telemetrics in |CL-ATTR| is a client and server solution used to collect data from running |CL| systems to help quickly identify and fix bugs in the OS.  Both client and server are customizable, and an API is available on the client side for instrumenting your code for debug and analysis. Note that the telemetry client is disabled by default, this is an opt-in solution.
```

Comment:
suggest to bold opt-in

Reply:
Please put it at the top.

E.g. something like.

```
Telemetry in |CL| is *opt-in*. By default, it will not be active or send any data. You need to explicitly enable it before it becomes active.
```

We can then elaborate on it later. :)

Reply:
added an "important" admonition above the note about privacy policies, following first paragraph.  This should draw the eye.